Plugins/MeshFeatureTracking/src/MeshFeatureTrackingComponent.cpp

```diff
@@ -582,30 +582,30 @@ Scalar MeshFeatureTrackingComponent::getFeatureScale() const {
     // manage picking mode
     auto ro = getRoMgr()->getRenderObject( m_pickedRoIdx );
     if ( m_pickedMesh->getRenderMode() == MeshRenderMode::RM_POINTS )
-    { return ro->getAabb().sizes().norm() / 500; }
+    { return ro->getAabb().sizes().norm() / 500; } const auto& T = ro->getTransform();
```

Comment:
There is a format probleme here, please revert, or use clang-format script with up to date clang-format version

Reply:
Yes, I need to rebase so this will be done by the time